Packages/Timeline/Sources/Timeline/PendingStatusesObserver.swift

```diff
@@ -1,51 +1,86 @@
 import Foundation
 import SwiftUI
 import Models
+import DesignSystem
+import Combine
 
 @MainActor
 class PendingStatusesObserver: ObservableObject {
-  let feedbackGenerator = UIImpactFeedbackGenerator(style: .light)
+  @AppStorage("expanded") public var isExpanded: Bool = false
   
-  @Published var pendingStatusesCount: Int = 0
+  @Published var pendingStatuses: [Status] = []
```

Comment:
Note sure we should store the whole status here. I've made it an hint so the diffing of SwiftUI could be very fast. It was slow as yous scrolled up prior to that. 
I'm ok to have a shadow non published variable holding status tho. And actually you had it, `pendingStatuses`

Reply:
Made pendingStatuses , none published now